parsl/executors/high_throughput/process_worker_pool.py

```diff
@@ -275,10 +301,12 @@ def worker(worker_id, pool_id, task_queue, result_queue, worker_queue):
     """
     start_file_logger('{}/{}/worker_{}.log'.format(args.logdir, pool_id, worker_id),
                       worker_id,
-                      level=logging.DEBUG if args.debug is True else logging.INFO)
+                      name="worker_log",
+                      level=logging.DEBUG if args.debug else logging.INFO)
 
     # Sync worker with master
     logger.info('Worker {} started'.format(worker_id))
+    logger.debug("Debug logging enabled")
```

Comment:
only output this if args.debug - there might be other third party loggers in place, and they'll show a "debug logging enabled" message even when it isn't.